deploy_oss.prolific

```diff
@@ -469,8 +472,11 @@ Add `-o operations/scale-to-one-az.yml` to the deploy command. This ops-file wil
 
 If you want to speed up the deploy, you can also add `-o operations/use-compiled-releases.yml`, to skip compiling all of your releases.
 
+We also recommend you use the `--non-interactive` flag. Normally, after initial preprocessing steps, BOSH will present the "manifest changes", differences between the currently-deployed manifest (currently nothing) and the deploying manifest. Without this flag, you prompted ~5 minutes after the `bosh deploy` command.
```

Comment:
I really like this addition. The last sentence seems like it's missing a word or two. I wonder if we can change
> Without this flag, you prompted ~5 minutes after the `bosh deploy` command.
To
> Without this flag, you'll be prompted ~5 minutes after the `bosh deploy` command to approve the manifest changes.

Reply:
Since this has been sitting for some time, I'm going to merge as-is and open a follow-up PR to fix this typo. Going forward we can do some thinking about how to better deal with PRs which are waiting for response.